### ISSUE TITLE
Set keyboard focus on graph/table when selecting an analysis

### ISSUE THREAD

**Issue by PatrickTasse** [theia-ide/theia-trace-extension#311]
I would suggest the following improvements:

- When clicking on an analysis in the Available Analyses list, in the selected trace tab, this analysis's graph/table should be given keyboard focus and show the focus blue border. The trace tab should be scrolled to make that analysis visible, whether it is being opened for the first time or if it was already created previously. If the visible page is already full, opening a new analysis currently is only noticed by the scrollbar thumb position/size being updated, as the new analysis is out of view.

- When clicking on a trace tab, the keyboard focus should be given to the first, or last selected, analysis's graph/table in the tab.

- When clicking on an analysis title in the trace tab, the keyboard focus should be given to its graph/table

**Comment by ebugden** [comment]
Related: #271

**Comment by hriday-panchasara** [comment]
Closed via #612
